storage/feed.go

```diff
@@ -80,9 +81,13 @@ func (s *Storage) CountFeeds(userID int64) int {
 
 // CountUserFeedsWithErrors returns the number of feeds with parsing errors that belong to the given user.
 func (s *Storage) CountUserFeedsWithErrors(userID int64) int {
+	pollingParsingErrorLimit := config.Opts.PollingParsingErrorLimit()
+	if pollingParsingErrorLimit <= 0 {
+		return 0
+	}
```

Comment:
It's not because you can remove the limit that you have to stop reporting the error count to users.